server/channels/app/limits.go

```diff
@@ -38,10 +23,17 @@ func (a *App) GetServerLimits() (*model.ServerLimits, *model.AppError) {
 		limits.MaxUsersLimit = maxUsersLimit
 		limits.MaxUsersHardLimit = maxUsersHardLimit
 	} else if license != nil && license.IsSeatCountEnforced && license.Features != nil && license.Features.Users != nil {
-		// Enforce license limits as required by the license with grace period.
+		// Enforce license limits as required by the license with configurable extra users.
 		licenseUserLimit := int64(*license.Features.Users)
 		limits.MaxUsersLimit = licenseUserLimit
-		limits.MaxUsersHardLimit = calculateGraceLimit(licenseUserLimit)
+
+		// Use ExtraUsers if configured, otherwise default to 0 (no extra users)
+		extraUsers := 0
+		if license.ExtraUsers != nil {
```

Comment:
We are checking if we are enforcing the limit elsewhere before we enforce this hard limit correct ?